templates/artist/artist_list.html

```diff
@@ -11,86 +11,87 @@
-                            <li><a href="?campaign-status={{ campaign_status }}&genre=All%20Genres&sort={{ order_by.slug }}"><h6>All Genres</h6></a></li>
-                        {% endif %}
-                        {% for genre in genres %}
-                            {% if genre != active_genre %}
-                                <li><a href="?campaign-status={{ campaign_status }}&genre={{ genre|urlencode }}&sort={{ order_by.slug }}"><h6>{{ genre }}</h6></a></li>
+    <div class="list-nav">
+        <div class="filter-sort-options center">
+            <ul class="dropdown menu" data-dropdown-menu data-options="disableHover:true; clickOpen:true;">
+                    <li class="nav-float"><h6>Show me</h6></li>
+                    <li class="is-dropdown-submenu-parent opens-right nav-float">
+                        <a href="#"><h6>{{ campaign_status }}</h6></a>
+                        <ul class="menu is-dropdown-submenu">
+                            {% for option in campaign_statuses %}
+                                {% if option != campaign_status %}
+                                    <li><a href="?campaign-status={{ option }}&genre={{ active_genre }}&sort={{ order_by.slug }}"><h6>{{ option }}</h6></a></li>
+                                {% endif %}
+                            {% endfor %}
+                        </ul>
+                    </li>
+                <li class="nav-float"><h6>artists in</h6></li>
+                {% if genres %}
+                    <li class="is-dropdown-submenu-parent opens-right nav-float">
+                        <a href="#"><h6>{{ active_genre }}</h6></a>
+                        <ul class="menu is-dropdown-submenu">
+                            {% if active_genre != 'All Genres' %}
+                                <li><a href="?campaign-status={{ campaign_status }}&genre=All%20Genres&sort={{ order_by.slug }}"><h6>All Genres</h6></a></li>
                             {% endif %}
-                        {% endfor %}
-                    </ul>
-                </li>
-            {% endif %}
-            <li>
-                <button class="button hollow location" id="location-dropdown-button" type="button" data-toggle="location-dropdown">
+                            {% for genre in genres %}
+                                {% if genre != active_genre %}
+                                    <li><a href="?campaign-status={{ campaign_status }}&genre={{ genre|urlencode }}&sort={{ order_by.slug }}"><h6>{{ genre }}</h6></a></li>
+                                {% endif %}
+                            {% endfor %}
+                        </ul>
+                    </li>
+                {% endif %}
+                <li class="nav-float">
+                    <button class="button hollow location" id="location-dropdown-button" type="button" data-toggle="location-dropdown">
                     <h6>{% if distance %}within {{ distance }} miles{% endif %}
```

Comment:
nit: Because `<button>` above was indented, this line should be too, since it's inside the button.
`</button>` below should be aligned with the open tag.